schemas/mqtt/journey.json

```diff
@@ -181,6 +181,16 @@
                     "maximum": 180.0000
                   }
                 }
+              },
+              "description": {
+                "$id": "#/properties/route/properties/stopPlaces/items/properties/description",
+                "type": "string",
+                "description": "Description for the stop"
+              },
+              "publicCode": {
+                "$id": "#/properties/route/properties/stopPlaces/items/properties/publicCode",
+                "type": "string",
+                "description": "Public code for a stop, also referred to as puck"
```

Comment:
I don't think we need to reference the visual representation of the public code. We call it a puck because we show the data in a circle that is like (presumably?) a hockey puck.